spec/features/users/user_profile_spec.rb

```diff
@@ -1,6 +1,29 @@
 require 'rails_helper'
 
 feature 'User profile' do
+  context 'account status badges' do
+    before do
+      sign_in_live_with_2fa(profile.user)
+    end
+
+    context 'LOA1 account' do
+      let(:profile) { create(:profile) }
+
+      it 'shows a "Basic Account" badge with a tooltip' do
+        expect(page).to have_content(t('headings.profile.basic_account'))
```

Comment:
Ok so I did that in f544bedbf4613dfd338bb546959c7ac67111f64d

However, I had to both mock out a user and assign an ivar separately, so my worry is that this may be an insufficient test in the future due to loose integration boundaries, where the feature spec was more "complete"?

I am partial to removing that commit and using the feature spec. What do you think?